.gitlab/pipeline/test.yml

```diff
@@ -619,3 +619,10 @@ subsystem-benchmark-approval-voting:
   script:
     - cargo bench -p polkadot-node-core-approval-voting --bench approval-voting-regression-bench --features subsystem-benchmarks
   allow_failure: true
+
+subsystem-benchmark-estimate-cpu-requirements:
+  extends:
+    - .subsystem-benchmark-template
+  script:
+    - cargo run -p polkadot-subsystem-bench  --release    -- polkadot/node/subsystem-bench/examples/estimate_cpu_requirements.yaml
```

Comment:
I'd remove these so we can merge.

Reply:
We don't need it in the CI, right? Is it here just to run on a reference hw?